dae-alright/src/components/Navigation/Navigation2.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "../Navigation/Navigation2.scss";
```

Comment:
Your navigation bar is missing some styling, kindly look into it

# Screenshot
![navbar](https://user-images.githubusercontent.com/46860081/68209678-33e05400-ffd4-11e9-8aad-aec8e0e14dc1.png)